exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/QueryWrapper.java

```diff
@@ -44,7 +44,7 @@
 public class QueryWrapper {
   private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(QueryWrapper.class);
   // Heap usage threshold/trigger to provide resiliency on web server for queries submitted via HTTP
-  private static final double HEAP_MEMORY_FAILURE_THRESHOLD = 0.85;
+  private double memoryFailureThreshold = 0.85;
```

Comment:
There is no need to keep it as class variable, use re-assign it each time in run method.

Reply:
+1